src/App.tsx

```diff
@@ -1,14 +1,63 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
```

Comment:
Now this state is useless. Anyway, you set loadedTodos to visibleTodos state in `getTodosFromServer`. So with such components and states structure better to remove it

Another way - implement App as Vlad suggested